packages/node_modules/@webex/http-core/test/integration/spec/interceptor.js

```diff
@@ -22,15 +22,33 @@ describe('http-core', () => {
       })));
 
     describe('logOptions', () => {
-      it('calls logger plugin', () => {
+      it('calls logger plugin if enabled', () => {
```

Comment:
In order to achieve the ["Arrange, Act, Assert"](https://defragdev.com/blog/?p=783), I'd prefer to see the setup of the env vars in a `beforeEach` block. Then returning the env var back in an `afterEach` block.